pkg/scheduler/framework/plugins/nodeaffinity/node_affinity_test.go

```
@@ -852,7 +853,13 @@ func TestNodeAffinity(t *testing.T) {
 			if err != nil {
 				t.Fatalf("Creating plugin: %v", err)
 			}
-			gotStatus := p.(framework.FilterPlugin).Filter(context.Background(), nil, test.pod, nodeInfo)
+
+			state := framework.NewCycleState()
+			var gotStatus *framework.Status
+			if !test.disablePreFilter {
+				_ = p.(framework.PreFilterPlugin).PreFilter(context.Background(), state, test.pod)
```

don't discard the error. Call `t.Errorf`